src/main/java/com/icthh/xm/ms/entity/config/WebConfigurer.java

```diff
@@ -138,14 +138,6 @@ public CorsFilter corsFilter() {
         return new CorsFilter(source);
     }
 
-    @Bean(name = "multipartResolver")
-    public CommonsMultipartResolver multipartResolver(ApplicationProperties applicationProperties) {
```

Comment:
did you tested work of multipart (!!! not json) request after this change?

Reply:
we are switching to default implementation/configuration